server/notification-providers/telegram.js

```diff
@@ -9,9 +10,17 @@ class Telegram extends NotificationProvider {
         let okMsg = "Sent Successfully.";
 
         try {
+            let message = msg;
```

Comment:
Not sure what the point of this is?

Reply:
This is added as it is recommended not to change the function attributes directly, so I am saving into another variable and changing it further by concatenating it with another string.